api/v2.0/swagger.yaml

```diff
@@ -20,6 +20,215 @@ security:
   - basic: []
   - {}
 paths:
+  /projects:
+    get:
+      summary: List projects
+      description: This endpoint returns all projects created by Harbor, and can be filtered by project name.
```

Comment:
Remove `and can be filtered by project name` as the result can be narrowed down by several filters, not only project name